.github/workflows/release.yml

```diff
@@ -57,16 +57,15 @@ jobs:
         run: |
           if [ -n "$RELEASE_VERSION" ]; then
             # For production release, we run on five python versions.
-            echo 'matrix={"include":[{"name":"taichi","python":"3.7"},{"name":"taichi","python":"3.8"},{"name":"taichi","python":"3.9"},{"name":"taichi","python":"3.10"},{"name":"taichi","python":"3.11"},{"name":"taichi","python":"3.12"}]}"' >> $GITHUB_OUTPUT
-
+            echo 'matrix={"include":[{"name":"taichi","python":"3.9"}, {"name":"taichi","python":"3.10"},{"name":"taichi","python":"3.11"},{"name":"taichi","python":"3.12"},{"name":"taichi","python":"3.13"}]}"' >> $GITHUB_OUTPUT
```

Comment:
here you are keeping 3.9. I feel if you are keeping 3.9 here, then 3.9 should be included in other grids etc, and should be the version of python we use for testing anything that is not gridded.